tox.ini

```diff
@@ -1,6 +1,7 @@
 [tox]
+package=os_api_cache
 envlist=
-  py35
+  py36
 skip_missing_interpreters = true
```

Comment:
Same small tweak, no point skipping missing interpreters if you just have one. However, it doesn't matter much anyway.